connect.js

```diff
@@ -138,6 +138,7 @@ const wrap = (Wrapped, module, epics, logger, options = {}) => {
           }
         }
       });
+      resources = null;
```

Comment:
`resources = []` maybe to take care of that failing test? 

I also wonder if `_.map(...)` on line 36 should be `_.each(...)` since we don't do anything with the return value from `map`. Probably doesn't make a big difference, but why make a copy of the array if we're just going to ignore it?

Reply:
Good catch @zburke. `forOwn` would probably work better here since it's an object.